pyscriptjs/tests/py-unit/pyscript_plugins_tester.py

```diff
@@ -0,0 +1,116 @@
+import xml.dom
+from xml.dom.minidom import Node  # nosec
+
+import pyscript
+
+
+class classList:
+    """Class that (lightly) emulates the behaviour of HTML Nodes ClassList"""
+
+    def __init__(self):
+        self._classes = []
+
+    def add(self, classname: str):
+        """Add classname to the classList"""
+        self._classes.append(classname)
+
+    def remove(self, classname: str):
+        """Remove classname from the classList"""
+        self._classes.remove(classname)
+
+
+class PluginsManager:
+    """
+    Emulator of PyScript PluginsManager that can be used to simulate plugins lifecycle events
```

Comment:
Following your note 2, maybe we can add a todo or a comment saying that its missing most of the lifecycle events and that it currently only works for python plugins? (I think?)